arenadata/run_docker.sh

```diff
@@ -1,7 +1,7 @@
 #!/usr/bin/env bash
 set -exo pipefail
 
-DOCKERIMAGE="${1:-hub.adsw.io/library/gpdb6_regress:latest}"
+DOCKERIMAGE="${1:-hub.adsw.io/library/gpdb6_u22:latest}"
```

Comment:
Should we also change image name in [Dockerfile](https://github.com/arenadata/pgbackrest/blob/39f72cc338a1c76ef70cfffe7ee4946300591c7c/arenadata/Dockerfile#L1C1-L2C1) (since it's used in `docker build` command below)? Could you please describe it?

Reply:
Sure, but the image name in Dockerfile is overwritten by this setting, so it doesn't affect anything.

> Could you please describe it?

Describe what exactly?

Reply:
> Describe what exactly?
You already answered. So it's ok.